MainTest.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class MainTest{
+
+@Test
+public void testNothing() {
+
+}
+
+@Test
+public void testSomeStuff() {  
+assertEquals("False", 5, 2+3);
+
+String test = new String("test");
+assertNotNull("Object is null", null);
```

Comment:
Hier hätte ich eigentlich folgendes erwartet:
`assertNotNull( "Object is null", new String("Test") );`